packages/libs/deer/rust-toolchain.toml

```diff
@@ -0,0 +1,4 @@
+[toolchain]
+# Please also update the badges in `README.md`, `src/lib.rs`, and `macros/`
+channel = "nightly-2022-09-27"
```

Comment:
Do we plan to also support the stable channel? If yes, can we add this crate here as well?
https://github.com/hashintel/hash/blob/8567e5a77a8fdae61d4398c8ac1bcde7a6ea5828/.github/scripts/rust/setup.py#L25-L27

Reply:
wanted to do this once I setup CI in another PR, for now I just want to get the project skeleton in order

Reply:
Corresponding CI PR: https://github.com/hashintel/hash/pull/1164